### PR TITLE
[fix](fuzzy) make fuzzy config of BE really take effect

### PR DESCRIPTION
**Pull request by zclllyybb**
# Proposed changes

## Problem summary

Before, our fuzzy configs of BE are not really random --- It generates the same configs every time. Now fix it.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [x] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zclllyybb** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 34.13 seconds
 stream load tsv:          425 seconds loaded 74807831229 Bytes, about 167 MB/s
 stream load json:         23 seconds loaded 2358488459 Bytes, about 97 MB/s
 stream load orc:          60 seconds loaded 1101869774 Bytes, about 17 MB/s
 stream load parquet:          30 seconds loaded 861443392 Bytes, about 27 MB/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230515091234_clickbench_pr_143733.html

**Comment by zclllyybb** [comment]
run arm

**Comment by zclllyybb** [comment]
run buildall

**Comment by dataroaring** [comment]
run buildall

**Comment by zclllyybb** [comment]
run buildall

File: be/src/common/config.cpp [hunk 1]
```diff
@@ -1536,13 +1538,14 @@ Status set_fuzzy_config(const std::string& field, const std::string& value) {
 
 void set_fuzzy_configs() {
     // random value true or false
-    static_cast<void>(
-            set_fuzzy_config("disable_storage_page_cache", ((rand() % 2) == 0) ? "true" : "false"));
-    static_cast<void>(
-            set_fuzzy_config("enable_system_metrics", ((rand() % 2) == 0) ? "true" : "false"));
+    Random generator {(uint32_t)std::time(nullptr)};
+    static_cast<void>(set_fuzzy_config("disable_storage_page_cache",
```
Comment (BiteTheDDDDt) [inline be/src/common/config.cpp:1542]: remove static_cast<void>

**Pull request closed without merging**